### ISSUE TITLE
Docker

### ISSUE THREAD

**Issue by jeanfrancis** [etsy/opsweekly#42]
Any dockerfile available to test Opsweekly ?

**Comment by sammcj** [comment]
Good idea - I'd write spin one up myself and submit a PR, but it looks to currently rely on MySQL which I'm not keen on working with as we've finally decommissioned our last MySQL server and I want it to stay that way.

If this changes, I'd be happy to setup docker-opsweekly.

**Comment by jeanfrancis** [comment]
i don't think is limited to MySql see this http://docs.docker.com/examples/postgresql_service/

**Comment by sammcj** [comment]
@jeanfrancis It's not Docker that's limited to MySQL, it's OpsWeekly

**Comment by ninoabbate** [comment]
If still interested I created a dockerized build of Opsweekly: https://github.com/ninoabbate/docker-opsweekly
